README.md

```diff
@@ -5,10 +5,19 @@
 ## Prerequisites
 
 * Golang (1.10)
+ * [Dep tool](https://golang.github.io/dep/docs/installation.html)
+* [oc tools >= 3.9.0](https://github.com/openshift/origin/releases)
 * Nodejs
```

Comment:
Do we know if there's a minimum version we require? Maybe we could link out to the install guides for Go and Node too?

Reply:
The answer to "What is the minimum number we require" is "What is the minimum we want to support".

Reply:
Good question. Not one I know the answer to. But as Node 8 is the current active LTS release that could be a safe bet for now? https://github.com/nodejs/Release#release-schedule